### ISSUE TITLE
after alt-text audit done, the accessibility issue template should update to allow missing alt-text reports.

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3839]
A11y template created requesting it not be used for Alt-text at present.  If multiple people were reporting missing alt-text there was a high chance of duplication.

But once the audit is complete and all those missing alt-texts have been updated, then this should be part of the A11y issue template so that any future missing alt-text is reported.

**Comment by StephDriver** [comment]
the issue template has been updated.
